Tasks/Gordeenko/Task3/Task3_1.py

```diff
@@ -0,0 +1,25 @@
+# Написать программу для поиска наиболее часто встречающихся элементов в строке.
+# Пользователь вводит строку, и количество элементов, которые он хотел бы увидеть в ответе.
+# Строка:lkseropewdssafsdfafkpwe
+
+x = list(input("Please enter text:"))
+
+#x = list("lkseropewdssafsd fafk1pwe")
+
+d = {i : x.count(i) for i in x}
+
+sorted_values = sorted(d.values())[::-1]
+sorted_d = {}
+
+for i in sorted_values:
+    for k in d.keys():
+        if d[k] == i:
+           sorted_d[k] =d[k] 
```

Comment:
this is not very good approach, despite its working alright, because it basically breaks the dictionary structure and then creates it again, but all the key-value pairs would be added several times (n times if we have n equal values in the initial dictionary for n different keys)